HIP XX Crowdspot Modifications

```diff
@@ -0,0 +1,190 @@
+No other wallet shall be used for locking the staking bond other than the wallet currently linked to Crowdspot. 
+The total Staked Bond will be a product of the number hotspots included on the submission multiplied by var_staked_bond_amount HNT. 
+A mechanism shall be put in place to route any appeal submission to a different member of the review team than it's initial submission. 
+Due Notice of change shall be sent to the hotspot owner regarding changes to their denylist status and/or changes to their submission status. 
+At the time a hotspot address is being entered on a removal submission form, the data must be validated against the denylist and Crowdspot to determine the appropriate type of removal request to be labeled on the submission.  Initial removal submissions are permitted for hotspots that were denied via algorithmic detection and have not had a prior addition submission that was "accepted" by the review team.   Appeal removal submissions are permitted for hotspots that have a prior addition submission "accepted" by the review team  or have a prior initial removal submission as "declined" by the review team.   
+In the event that an owner attempts to add multiple hotspots to a submission that are of mixed "initial" and "appeal" types, the form data shall not be validated and an error message shall alert the owner to try one hotspot per submission.  
+Only one initial removal submission and one appeal removal submission on a particular hotspot can ever be worked by the review team.  Once a determination of "declined" has been decided for an appeal submission, no removal submission can be accepted for that particular hotspot.  
+
+
+
+ Staked Bond in detail:
+
+     The staked bond amount shall be var_staked_bond_amount HNT per hotspot. (This was intended to be approximately equivalent to the average network earnings of one hotspot for var_submission_expiration_time days.)  Only one staked bond shall be locked up for the initial and appeal removal submission.  The staked bond shall be carried over from an initial removal submission into an appeal removal submission provided that the owner creates the appeal removal submission within var_submission_expiration_time days of the denial determination by the review team.   
+
+   The staked bond shall be immediately returned to the owner's wallet under any one of these three conditions:  
+ Any removal submission is approved by the review team and POC gaming was not substantiated. 
+An Owner Cancelation is executed on an initial removal submission prior to a determination by the review team.  Appeal removal submission stake bonds can not be recovered by owner cancellation. See OWNER DEADLINES for details.  
+The var_submission_expiration_time day time limit for any submission to be reviewed has expired. See REVIEW TEAM DEADLINES for details.  
+
+  
+
+The staked bond shall be forfeited and burned under either of these two conditions:
```

Comment:
Potentially change burning to placing the money into at treasury to be paid to falsepositive hotspot owners

Reply:
Note: This is the same premises as PR551